server/src/main/java/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -1049,7 +1049,7 @@ private Type getSnapshotType(IntervalType intvType) {
         return null;
     }
 
-    private boolean hostSupportSnapsthotForVolume(HostVO host, VolumeInfo volume) {
+    private boolean hostSupportSnapsthotForVolume(HostVO host, VolumeInfo volume, boolean isFromVmSnapshot) {
```

Comment:
can you correct the type error in method name

```suggestion
    private boolean hostSupportsSnapshotForVolume(HostVO host, VolumeInfo volume, boolean isFromVmSnapshot) {
```

Reply:
good suggestion. @ravening could you change it ?

Reply:
@sureshanaparti made the change. can you review again?